Terminal.Gui/Input/Responder.cs

```diff
@@ -47,6 +47,11 @@ public Responder ()
 	}
 #endif
 
+	/// <summary>
+	/// Invokes subscribers that this is disposed.
```

Comment:
`/// Event raised when <see cref="Dispose"/> is called to signal that this object has been disposed.`